card/lib/card.rb

```diff
@@ -4,20 +4,10 @@
 
 class Card < ActiveRecord::Base
   require_dependency 'card/active_record_ext'
-  require_dependency 'card/codename'
   require_dependency 'card/query'
-  require_dependency 'card/set_pattern'
-  require_dependency 'card/set'
-  require_dependency 'card/format'
   require_dependency 'card/exceptions'
-  require_dependency 'card/auth'
   require_dependency 'card/log'
-  require_dependency 'card/loader'
-  require_dependency 'card/content'
   require_dependency 'card/action'
-  require_dependency 'card/act'
-  require_dependency 'card/change'
-  require_dependency 'card/reference'
```

Comment:
It occurs to me that one nice thing about having these explicit is that it hedges against cases where users define a root level constant of the same name.

For example, let's say I create a rails app with a module in the root namespace named Change (or Act or Reference or Format or whatever).

Then I add decko-rails to my app.

When there is a contextual reference to Card::Change like this:

````
class Card
  module MyMod
    def all_changes
      Change.all
    end
  end
end
````
...that only works with autoloading so long as there is no known constant "Change".

So, after further thought, I think we should probably keep these explicit and *only* autoload card.rb itself.